src/Effect/Exec.hs

```diff
@@ -440,13 +437,10 @@ selectBestCmd workdir CandidateAnalysisCommands{..} = selectBestCmd' (NE.toList
 
 data CandidateCommandFailed = CandidateCommandFailed {failedCommand :: Text, failedArgs :: [Text]}
 instance ToDiagnostic CandidateCommandFailed where
-  renderDiagnostic CandidateCommandFailed{..} =
-    pretty $
-      "Command "
-        <> show failedCommand
-        <> " not suitable: running with args "
-        <> show failedArgs
-        <> " resulted in a non-zero exit code"
+  renderDiagnostic CandidateCommandFailed{..} = do
+    let header = "Command: " <> failedCommand <> " not suitable"
```

Comment:
It would be nice to surround the failedCommand and the failedArgs with quotes or put them on a newline so that we more clearly distinguish the command and/or args from the help text. This is out of scope for this ticket, as it's an improvement to an existing error message, but I thought I'd mention it while I noticed it